gamemode/languages/sh_spanish.lua

```diff
@@ -78,37 +135,40 @@ LANGUAGE = {
 	dSetFaction = "Ésta puerta ahora pertenece a la facción %s.",
 	dRemoveFaction = "Ésta puerta ya no pertenece a ninguna facción.",
 	dNotValid = "No estás mirando a una puerta válida.",
-	canNotAfford = "No puedes permitirte el gasto para comprar ésto.",
+	canNotAfford = "No puedes permitirte el gasto para comprar esto.",
 	dPurchased = "Has comprado ésta puerta por %s.",
 	dSold = "Has vendido ésta puerta por %s.",
-	notOwner = "No eres el propietario de ésto.",
+	notOwner = "No eres el propietario de esto.",
 	invalidArg = "Has provisto un valor inválido para el argumento #%s.",
+	invalidFaction = "La facción que has provisto es inválida.",
 	flagGive = "%s ha dado a %s las flags '%s'.",
 	flagGiveTitle = "Dar Flags",
-	flagGiveDesc = "Da las siguientes flags al jugador.",
+	-- flagGiveDesc = "Da las siguientes flags al jugador.",
 	flagTake = "%s ha quitado las flags '%s' de %s.",
 	flagTakeTitle = "Quitar Flags",
-	flagTakeDesc = "Quita las siguientes flags del jugador.",
+	-- flagTakeDesc = "Quita las siguientes flags del jugador.",
 	flagNoMatch = "Debes tener la(s) flag(s) \"%s\" para hacer ésta acción.",
 	textAdded = "Has añadido un texto.",
 	textRemoved = "Has borrado %s texto(s).",
 	moneyTaken = "Has encontrado %s.",
+	moneyGiven = "Te han dado %s.",
+	insufficientMoney = "No tienes dinero suficiente para hacer esto.",
 	businessPurchase = "Has comprado %s por %s.",
 	businessSell = "Has vendido %s por %s.",
+	businessTooFast = "Por favor, espera antes de comprar otro objeto.",
 	cChangeModel = "%s ha cambiado el modelo de %s a %s.",
 	cChangeName = "%s ha cambiado el nombre de %s a %s.",
 	cChangeSkin = "%s ha cambiado el skin de %s a %s.",
```

Comment:
I wouldn't change the word skin in this case, as bodygroup, due to both being specific words that have to do with model properties.

Reply:
I changed it since there's a specific word for it, and it changes the general look of the model itself, in most cases. Though it could stay as skin. Whatever works I guess.